### ISSUE TITLE
knctl build create - blocks/never exits with standalone Knative Build

### ISSUE THREAD

**Issue by drnic** [cppforlife/knctl#33]
I tried installing Knative Build without Knative Serving/Istio, and `knctl build create` blocks at the end. From the `kail -n knative-build` logs, I think it might be blocking because no deploying is waiting for it:

```
{"level":"error","logger":"controller","caller":"cluster/builder.go:278","msg":"Saw \"default/2-lgrln\" update, but nothing was watching for it!"
```

Reproduction steps at https://starkandwayne.com/blog/p/4884e933-6a0a-43a5-b794-2afaa140c8bc/

**Comment by cppforlife** [comment]
interesting, what does `knctl build show -b xxx` output?

**Comment by drnic** [comment]
@cppforlife the `show` output says `Succeeded` at the end.

**Comment by cppforlife** [comment]
looks like in later versions of knative, following

```
  conditions:
  - lastTransitionTime: 2018-11-06T01:02:26Z
    status: "True"
    type: Succeeded
```

cannot be parsed via

```
// BuildCondition defines a readiness condition for a Build.
// See: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#typical-status-properties
type BuildCondition struct {
	Type BuildConditionType `json:"state"`

	Status corev1.ConditionStatus `json:"status" description:"status of the condition, one of True, False, Unknown"`

	// +optional
	Reason string `json:"reason,omitempty" description:"one-word CamelCase reason for the condition's last transition"`
	// +optional
	Message string `json:"message,omitempty" description:"human-readable message indicating details about last transition"`
}
```

**Comment by cppforlife** [comment]
once knative 0.2.0 is release, knctl will be updated to use new apis. since upstream knative will not fix this issue, i m going to close this here. future version of knative will be more careful about changing apis.

**Comment by cppforlife** [comment]
decided to fix it as im updating knctl to knative 0.2.x: https://github.com/cppforlife/knctl/commit/c1a73dae7148ee2429a98040a7313d517188ec5d
